libs/sdk-ui-ext/src/internal/dashboardEmbedding/KpiContent/KpiContent.tsx

```diff
@@ -107,9 +114,21 @@ class KpiContent extends Component<IKpiContentProps & WrappedComponentProps> {
 
     render(): React.ReactNode {
         return (
-            <div className="gd-kpi-widget-content">
-                {this.renderValue()}
-                {this.renderPeriodOverPeriod()}
+            <div className="gd-drill-modal-wrapper-mask">
```

Comment:
Is the `gd-drill-modal-wrapper-mask` div really needed here? SDK has no knowledge of any drill modals, those are only in gdc-dashboards.

Reply:
Can change className, but need custom styles so the widget is displayed correctly when height customization is enabled.